contracts/bprotocol/avatar/Avatar.sol

```diff
@@ -64,4 +65,26 @@ contract Avatar is AbsComptroller, AbsCToken {
         uint256 result = super.mint(cToken, mintAmount);
         return result;
     }
+
+    // EMERGENCY FUNCTIONS
+    function resetApprove(IBToken bToken) external onlyAvatarOwner {
+        address cToken = bToken.cToken();
+        ICToken(cToken).underlying().safeApprove(cToken, 0);
+    }
+
+    function transferERC20(address token) external onlyAvatarOwner {
```

Comment:
with all trasnfer fee etc, maybe to be safe add `amount` param?
just in case it will not be possible to send everything.